pandas/core/window.py

```diff
@@ -899,11 +899,45 @@ def skew(self, **kwargs):
         return self._apply('roll_skew', 'skew',
                            check_minp=_require_min_periods(3), **kwargs)
 
-    _shared_docs['kurt'] = """Unbiased %(name)s kurtosis"""
+    _shared_docs['kurt'] = dedent("""Calculate unbiased %(name)s kurtosis.
 
-    def kurt(self, **kwargs):
+    This function uses Fisher's definition of kurtosis (kurtosis of normal
+    == 0.0) without bias.
+
+    Returns
+    -------
+    same type as input
+
+    See Also
+    --------
+    scipy.stats.kurtosis
+    pandas.DataFrame.kurtosis
+    pandas.Series.kurtosis
+
+    Notes
+    -----
+    A minimum of 4 periods is required for the rolling calculation
+
+    Examples
+    --------
+    >>> arr = [1, 2, 3, 4, 5]
+    >>> import scipy.stats
+    >>> scipy.stats.kurtosis(arr, bias=False)
+    -1.2000000000000004
+
+    >>> df = pd.DataFrame(arr)
+    >>> df.rolling(5).kurt()
+        0
+    0  NaN
+    1  NaN
+    2  NaN
+    3  NaN
+    4 -1.2
```

Comment:
Probably it's obvious enough, but may be having 6 values instead of 5, would give more the idea that after the 4th `NaN` all the values will be filled?